catalog-info.yaml

```
@@ -177,6 +177,39 @@ spec:
         search-extract-and-transform: {}
         search-productivity-team: {}
 
+# Dockerfiles nightly images build pipeline
+---
+apiVersion: "backstage.io/v1alpha1"
+kind: "Resource"
+metadata:
+  name: "connectors-dockerfiles-nightly"
+  description: "Nightly Connectors Dockerfiles Build"
```

```suggestion
  description: "Nightly build and scan of the public/extensible connectors Dockerfiles"
```

describing how these Dockerfiles are different to the 'official' (`.wolfi`) ones is sort of difficult

maybe there is a better descriptor than 'public/extensible'? but that's my best idea at this point